feature-group-definitions/array-isarray.yml

```
@@ -1,5 +1,5 @@
 name: Array.isArray()
-description: Array.isArray() static method
+description: The `Array.isArray()` static method TODO.
```

This breaks the rules of circular definition but it feels unreasonable to do otherwise.

```suggestion
description: The `Array.isArray()` static method checks whether a value is an array.
```